pkgs/applications/networking/instant-messengers/signal-cli/default.nix

```diff
@@ -10,16 +10,24 @@ stdenv.mkDerivation rec {
     sha256 = "1gvdifscyxmxn2dwlkqi684ahy5kbcj84mqda0m8l4aa8iaq1d59";
   };
 
-  buildInputs = [ makeWrapper ];
+  buildInputs = [ libmatthew_java dbus dbus_java ];
```

Comment:
This should fix evaluation on macOS:
```suggestion
```

It is not required on Linux as far as I can see?

Reply:
Yeah I somehow forgot to commit that one. Is fixed.